### PR TITLE
this.js file created

### PR DESCRIPTION
**Pull request by poorna1307**

### CODE REVIEW

**Comment by poorna1307** [comment]
I haven't yet completed working on this topic, I will continue again tomorrow.

**Comment by poorna1307** [comment]
I haven't completed working on this topic, I will continue again tomorrow.

File: this.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const person_this = {
+    firstName: "John",
+    lastName: "Doe",
+    id: 5566,
+    fullName : function() {
+      return this.firstName + " " + this.lastName;
+    }
+  };
```
Comment (hemanntthreddy) [inline this.js:8]: we can use this operator in lot of places based on the use case, please practice more on this operator

**Comment by hemanntthreddy** [comment]
> I haven't completed working on this topic, I will continue again tomorrow.

okay

File: this.js (hunk 1 above)
Comment (poorna1307) [reply this.js:8]: sure, I will.

**Pull request merged**